docs/content/docs/advanced/summarizer.md

```diff
@@ -3,8 +3,136 @@ title: Summarizer
 menuPosition: 2
 ---
 
-Summaries are client-generated snapshots of the state of the document at a given sequence number. It consolidates the op
-log by serializing the data model.
+Summaries are client-generated snapshots of the state of the document at a given sequence number. A summary is a
+consolidation of the op log into one JSON blob.
+
+## Why have summaries?
+
+Summaries allow new clients to quickly catch up to recent state.
+
+Without a summary, the client has to apply every operation in the op log even if those operations no longer affect the
+current state (e.g. op 1 inserts 'h' and op 2 deletes 'h').
+
+## What do I need to know?
+
+Ideally very little. If you are building on top of our runtime and dds, then you won't need to think about the summary
+at all.
+
+When users with significant latency use your application, they may see the state of the container at the summary and
+then see new operations get applied. This can be hidden by waiting for the Container to be fully connected or you can
+allow users to see recent changes.
+
+## Summary lifecycle
+
+1. The Summary Manager selects a client to summarize the state of the Container (details below)
+2. The selected client spawns a non-user runtime that will generate the summary.
```

Comment:
Might be worth mentioning this is an entire separate client in quorum + loaded container?